atat_internal_api.yaml

```diff
@@ -1920,7 +1920,7 @@ components:
       type: object
       allOf:
         - $ref: "#/components/schemas/BaseObject"
-        - $ref: '#/components/schemas/AppEnvAccess'
+        - $ref: "#/components/schemas/AppEnvAccess"
```

Comment:
We seem to have this same pattern in a number of places near `$ref`s

Reply:
Yeah, that is true. I just happened to see that one. I reverted it back. I see that there are a good amount with both `' '` and `" "` being used. I figured since we will probably be making updates to the spec anyway that i'll leave it as is for now and only change the adding of the function to use.